Slambook/index.js

```diff
@@ -9,16 +9,46 @@ console.log(form,"form");
 // Add the evenlistner below
 form.addEventListener('submit',function(e){
     e.preventDefault();
-    console.log(document.querySelector('[name="contactName"]').value);
-    console.log(document.querySelector('[name="relationship"]').value);
-    console.log(document.querySelector('[name="likeInMe"]').value);
-    console.log(document.querySelector('[name="hateInMe"]').value);
-    console.log(document.querySelector('[name="deadReaction"]').value);
-    console.log(document.querySelector('[name="firstImpression"]').value);
-    console.log(document.querySelector('[name="beautifulMessage"]').value);
-    console.log(document.querySelector('[name="nickname"]').value);
-    console.log(document.querySelector('[name="songForMe"]').value);
-    console.log(document.querySelector('[name="shareOpinion"]').value);
+    const contactName = document.querySelector('[name="contactName"]')
+    const relationship = document.querySelector('[name="relationship"]')
+    const likeInMe = document.querySelector('[name="likeInMe"]')
+    const hateInMe = document.querySelector('[name="hateInMe"]')
+    const deadReaction = document.querySelector('[name="deadReaction"]')
+    const firstImpression = document.querySelector('[name="firstImpression"]')
+    const beautifulMessage = document.querySelector('[name="beautifulMessage"]')
+    const nickname = document.querySelector('[name="nickname"]')
```

Comment:
The function name 'removeSpan' does not follow the recommended naming convention. It should be 'removeSpanStyle'.